config/samples/config.yaml

```diff
@@ -0,0 +1,12 @@
+servers:
```

Comment:
not sure if you want to keep this but if so might want to add the hostname field

Reply:
updated